packages/toolpad-app/src/server/functionsTypesWorker.ts

```diff
@@ -475,15 +475,17 @@ export default async function extractTypes({
     .filter(Boolean)
     .sort((a, b) => a.name.localeCompare(b.name));
 
-  console.warn(
-    `${chalk.yellow('warn')} - ${chalk.bold(
-      usingCreateFunction.length,
-    )} function are using the deprecated ${chalk.red(
-      'createFunction',
-    )} API. This will be removed from Toolpad in a future release. Please see ${chalk.underline(
-      chalk.blue('https://mui.com/toolpad/reference/api/create-function/'),
-    )} for migration information and updates.`,
-  );
+  if (usingCreateFunction.length > 0) {
```

Comment:
We could call it just once during the lifetime of the process, just like [`util.deprecate'](https://nodejs.org/api/util.html#utildeprecatefn-msg-code) does.

Reply:
By storing a flag to check if the warning has been sent?

Reply:
Yes, unless `util.deprecate` already does the job reasonably well

Reply:
I get https://vitejs.dev/guide/troubleshooting.html#module-externalized-for-browser-compatibility when trying to use `util` to wrap `createFunction`. 

Another benefit is being able to pinpoint how many functions are using the deprecated function, which is a helpful addition to the warning in my view